common/src/main/java/com/wynntils/mc/mixin/ItemStackMixin.java

```diff
@@ -33,6 +36,29 @@ private Component redirectGetHoveredName(ItemStack instance) {
         return event.getHoveredName();
     }
 
+    @Redirect(
```

Comment:
I plan to start using https://github.com/LlamaLad7/MixinExtras/ soon, it will improve mixin compatibility greatly. These 2 mixins are a good candidate for their first time use. `LlamaLad7`, the developer, is currently working on adding a way to access any local variable in the current frame, working with any injection type.

Reply:
I looked at it, and I agree, it seem nice. I'll push these as-is, and then you can convert them to "mixinextras".